customer_satisfaction/steps/clean_data.py

```diff
@@ -0,0 +1,22 @@
+from zenml.steps import step, Output
+from model.data_cleaning import DataCleaning
+import pandas as pd
+
+
+@step
+def clean_data(
+    data: pd.DataFrame,
+) -> Output(
+    x_train=pd.DataFrame,
+    x_test=pd.DataFrame,
+    y_train=pd.Series,
+    y_test=pd.Series,
+):
+    """

```

Comment:
Add a short description to the docstring that explains what cleaning data means here